yarn.lock

```diff
@@ -1658,6 +1660,11 @@
     react-focus-lock "^2.2.1"
     timeago.js "^4.0.2"
 
+"@guardian/libs@^1":
```

Comment:
This seems regrettable. Would it be worth releasing a new version of CMP with @guardian/libs ^2 as a peer dependency [rather than ^1](https://github.com/guardian/consent-management-platform/blob/2864b35ba4934fec7ec289c2db266e0414ee51d3/package.json#L98)

Reply:
Yes, definitely! The bits that are included are very minimal anyways, and so far, the dependencies were bundled anyways, because rollup wasn’t smart enough to remove it.

Reply:
Ok great, I'm including libs 1 -> 2 in https://github.com/guardian/consent-management-platform/pull/474